proto/proto/responses.proto

```diff
@@ -61,7 +62,7 @@ message AccountBlockInputRecord {
 // A nullifier returned as a response to the GetBlockInputs
 message NullifierBlockInputRecord {
     digest.Digest nullifier = 1;
-    merkle.MerklePath proof = 2;
+    smt.SmtOpening opening = 2;
```

Comment:
Instead of creating a separate message for `SmtOpening` would it be simpler to just serialize it as bytes (using our `Serializabe`/`Deserializable` interfaces)?

Reply:
I personally prefer using protobuf types instead of `bytes`; for readability, and extra type safety (attempting to deserialize into anything other than an `SmtProof` will cause a compile-time error).

What do you see as advantages of using `bytes` instead?

Reply:
Mostly just relative simplicity (given that we already have tested serialization code), and potentially easier maintainability - e.g., if we switch from Merkle paths to compact Merkle path, the messages still remain the same. Though, this could be a double-edged sword.

Not a strong preference from my side though.

Reply:
I will leave as is (for above-mentioned reasons), and if it becomes a pain, we can switch to `bytes`. But my fear is that we end up doing that in multiple `proto` files, up to a point where they barely convey any useful information (due to the excessive use of `bytes`).